datasafe-rest-impl/pom.xml

```diff
@@ -116,6 +116,16 @@
             <artifactId>spring-boot-starter-security</artifactId>
             <version>${spring-boot.version}</version>
         </dependency>
+        <dependency>
+            <groupId>de.adorsys</groupId>
+            <artifactId>datasafe-storage-impl-db</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
```

Comment:
We have mysql DB spinning in openshift, why h2? (it is ok only for tests with scope test)